Front page, single page and blog list formating.

gouten5010 opened the pull request:
Fixes #25, #27

mahirock committed:
Update .gitignore

mahirock committed:
Update img.
Update style.

mahirock committed:
Update style

mahirock committed:
Update style.

mahirock committed:
Update style.

mahirock committed:
Update style.

Commit:
Update front news style.

mahirock committed:
Update style.

mahirock committed:
Update page visual.

mahirock committed:
Update style.

mahirock committed:
Update style.

mahirock committed:
Update style.

mahirock committed:
Update style.

cb-okashita committed:
Update .gitignore

cb-okashita committed:
Update .entry-content first element style.

cb-okashita committed:
Reformat code.

chiilog committed:
avater position change

chiilog committed:
article padding and sidebar padding change

gouten5010 commented:
@mellowchanter @mahirock 
今度は2列で…
<img width="1042" alt="2016-05-18 13 59 07" src="https://cloud.githubusercontent.com/assets/3461188/15347902/b9af6d6a-1d00-11e6-8153-da26a24f6141.png">

mahirock committed:
Wapuu style modification.
Front-menu padding change.

gouten5010 commented:
- Sidebar space OK.
- .entry-content padding and margin style OK.

mahirock committed:
Entry-footer style modification.
Front-menu style modification.

mahirock commented:
@gouten5010 
entry-footerのテキストは、投稿者名がavatar imgとセットなので…
1列にする方法がちょっとわからなくて、こんな感じになってます。
すみませんが、良い方法があれば、教えてもらえると助かります！

gouten5010 commented:
entryのヘッダーの長さが短いです
<img width="728" alt="2016-05-18 14 22 32" src="https://cloud.githubusercontent.com/assets/3461188/15348644/6ec10ab4-1d07-11e6-9363-a6e4aa4a22de.png">

gouten5010 commented:
フロントページの記事のpadding-top、必要ありません
<img width="753" alt="2016-05-18 14 38 12" src="https://cloud.githubusercontent.com/assets/3461188/15348668/9698e322-1d07-11e6-919d-df0aaae077fd.png">

gouten5010 commented:
フロントページのメタ情報いりません
![welcome to the vccw hello vccw](https://cloud.githubusercontent.com/assets/3461188/15348792/5f61b298-1d08-11e6-8ede-374b4adebd8f.png)

mahirock committed:
Breakpoint change.
ul margin style modification.
li height change.

gouten5010 commented:
フロントページのシェアエリアが擬似コンテンツエリアより大きいです
<img width="722" alt="2016-05-18 14 57 02" src="https://cloud.githubusercontent.com/assets/3461188/15348851/d2efecc0-1d08-11e6-9f4e-dabf8d20c49d.png">

gouten5010 commented:
シェアエリア、アバターエリアのborder-bottomいりません
<img width="771" alt="2016-05-18 15 02 35" src="https://cloud.githubusercontent.com/assets/3461188/15348923/97d893ac-1d09-11e6-9f10-47a3aa6c51b3.png">

gouten5010 commented:
- Wappu OK

gouten5010 commented:
あとcssはPushしないで…

gouten5010 commented:
- Front menu OK.

mahirock commented:
comment28

@gouten5010 
私のローカルでは短くないのですが…どうしたら？

mahirock committed:
Sharedaddy border style delete.
front-page padding delete.

mahirock committed:
Avatar area border delete.

gouten5010 commented:
ブレークポイントで1Columnになった時点で短いですよ？
<img width="852" alt="2016-05-18 15 34 01" src="https://cloud.githubusercontent.com/assets/3461188/15349516/65d55732-1d0e-11e6-9aba-840f57562944.png">

gouten5010 commented:
- front page padding-top OK

gouten5010 commented:
- dalete border-bottom : share-area and meta box border OK.  
  @mahirock `.sharedaddy:last-child:after`を`display: none;`にしてください

mahirock committed:
Entry-header margin change.
sharedaddy &:last-child:after style change.
page entry-footer delete.
1column layout style change.
avatar size change.

mahirock commented:
後、
フロントページのシェアエリアが擬似コンテンツエリアより大きいです

これ残ってます！
以上かな…。

gouten5010 commented:
@mahirock ではアバターエリアは画像の大きさの修正も込みでこちらで巻き取りですね。

mahirock commented:
@gouten5010 
よろしくお願いします。

あ、これも含めてお願いできるのでしょうか？

gouten5010 commented:
@mahirock 
はい。フロントでメニュー生成しているために中にもうひとつコンテンツエリアをつくっているので、そのぶんmarginまたはpaddingが必要になります。

mahirock commented:
@gouten5010 あぁ、なるほど！
やります！

gouten5010 commented:
・avatar image size OK

cb-okashita committed:
Change avatar style.

mahirock committed:
Entry-content padding change.

mahirock commented:
@gouten5010 
シェアエリアは独自クラス【front-post-wrap】の外であってますよね？

gouten5010 commented:
@mahirock 合ってます

cb-okashita committed:
Fixes avatar style.

cb-okashita committed:
Fixes avatar style for SP.

mahirock commented:
@gouten5010 
私、あと、何やり残してますか？

gouten5010 commented:
https://github.com/wckansai2016/site-design/pull/49#issuecomment-219932411
@mahirock これ終わってますか？

mahirock commented:
@gouten5010 
これまだでした、すみません！

cb-okashita committed:
Reformat code.

mahirock committed:
Sharedaddy margin change.

mahirock commented:
margin追加してます。
これでOKでしょうか？

gouten5010 commented:
- Share area margin OK

gouten5010 commented:
close #27
